src/renderer/helpers/walletHelper.test.ts

```diff
@@ -46,18 +45,18 @@ describe('walletHelper', () => {
   })
   const BNB_WB: WalletBalance = mockWalletBalance({
     amount: assetToBase(assetAmount(4)),
-    walletAddress: 'bnb-address',
-    asset: AssetBNB
+    walletAddress: 'bsc-address',
+    asset: AssetBSC
   })
   const BUSD_WB: WalletBalance = mockWalletBalance({
     amount: assetToBase(assetAmount(4.1)),
-    walletAddress: 'busd-address',
-    asset: AssetBUSD74E
+    walletAddress: 'busdc-address',
```

Comment:
busdc-address << nit pick but what is this

Reply:
It is correct. Also BUSD needs to be updated as that doesn't exist either.

Reply:
but why name it busdc?